solver/src/liquidity/offchain_orderbook.rs

```diff
@@ -28,6 +33,23 @@ struct OrderSettlementHandler {
     order: Order,
 }
 
+fn inflight_order_filter(order: Order, inflight_trades: &HashSet<OrderUid>) -> Option<Order> {
```

Comment:
The signature is already designed to later on mutate partially fillable orders? Otherwise a simple bool filter would suffice (but it makes sense to already set this up like that).

Reply:
Yes, this was the purpose of the function signature. Good eye!